### ISSUE TITLE
How to create a formAssociated custom element with Solidjs

### ISSUE THREAD

**Issue by DynamicArray** [ryansolid/component-register#15]
I'm new to custom elements but, from what I've read, it would be within this wrapper that the formAssociated setting would need to be made.

Are there plans to update this library to support that?

Regards
John

**Comment by ryansolid** [comment]
Hmm.. yeah I've never seen that before. They probably added it to the spec in the past few years. The way it works does suggest we'd need to make this an option for the wrapper.

**Comment by DynamicArray** [comment]
Likely to be something that happens soon? I'd have a crack at a patch myself, but I think it might be beyond my skill set.

I've got a workaround for now, that creates a corresponding input in the owner document, but it's untidy...

**Comment by Sleepful** [comment]
yeah this is rather important, access to internals https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/attachInternals is equally important, without these there's little reason to not use something like Lit instead
